nginx/default.conf.template

```diff
@@ -0,0 +1,67 @@
+server {
+  listen       80;
+  listen  [::]:80;
+  server_name  localhost;
+
+  root /usr/share/nginx/html;
+  index index.html;
+
+  error_page 500 502 503 504 /50x.html;
+
+  gzip on;
+  gzip_disable "msie6";
+  gzip_vary on;
+  gzip_min_length 1000;
+  gzip_types text/plain text/html text/css application/javascript application/json;
+
+
+  # SPA routing, everything falls back to the root index.html
+  location / {
+    try_files $uri $uri/ /index.html =404;
+
+    add_header Cache-Control "no-store";
+    add_header X-Frame-Options "DENY" always;
```

Comment:
X-Frame-Options is superseded by frame-ancestors csp directive. Which is fine, but safari <=12 wrongly prioritises X-Frame-Options which is a DENY here, so safari 12 won't allow this to be embedded inside a iframe which is a problem if we support this ancient safari. Which I suppose we don't.